File: Knitted_Toys_Store.API/Contracts/OrderResponce.cs
using Knitted_Toys_Store.Domain.Models.Domain;

namespace Knitted_Toys_Store.API.Contracts
{
    public record OrderResponce(
        Guid Id,
        DateTime OrderDate,
        decimal TotalAmount,
        OrderStatus Status,
        string SurnameCustomer,
        string NameCustomer,
        string PhoneNumber,
        string Email,
        string DeliveryAddress,
        string DeliveryNotes);
}

File: Frontend/knitted_toys_store/app/types/Order/OrderResponce.ts
import { OrderStatus } from "../../Models/Order";
import { OrderItemsResponse } from "../OrderItems/OrderItemsResponse";


export interface OrderResponse{
    id?: string;
    orderDate?: Date;
    totalAmount?: number;
    status?: OrderStatus;
    surnameCustomer: string;
    nameCustomer: string;
    phoneNumber: string;
    email: string;
    deliveryAddress: string;
    deliveryNotes: string;
    orderItems?: OrderItemsResponse[];

    orderItemsResponse: OrderItemsResponse[];
}